doc/source/ray-overview/installation.rst

```diff
@@ -58,7 +58,7 @@ You can install the nightly Ray wheels via the following links. These daily rele
 
 .. note::
 
-  Python 3.10 support is currently experimental.
+  Python 3.9/3.10 support is currently experimental.
```

Comment:
Sorry I may miss the context here. Why is 3.9 experimental?

Reply:
see https://github.com/ray-project/ray/pull/21221#issuecomment-1196594716

Reply:
It's experimental for the same reason 3.10 is. Both don't build ray ml containers.

Reply:
@acxz do you happen to know why we don't build ray ml container (is it just because we didn't do or fundamentally impossible due to some dep issues?)

Reply:
It's due to this issue right here: https://github.com/ray-project/ray/issues/16599

Which I believe is solved, once this PR is merged in, I'll be testing py39 and py310 for the ray ml containers.

To directly answer your question, it was because of a dep issue, but I think those deps should have fixed themselves by now.